RecoTracker/Configuration/python/customiseForRunI.py

```diff
@@ -33,11 +46,59 @@ def customiseForRunI(process):
     del process.MixedTripletStep
     del process.PixelLessStep
     del process.TobTecStep
+    del process.JetCoreRegionalStep
 
     # Load the new Iterative Tracking configuration
     process.load("RecoTracker.Configuration.RecoTrackerRunI_cff")
 
     process.globalreco.insert(tgrIndex, process.trackingGlobalReco)
-    process.reconstruction_fromRECO.insert(tgrIndexFromReco, process.trackingGlobalReco)
+    process.globalreco.insert(tgrIndex, process.recopixelvertexing)
+    process.reconstruction_fromRECO.insert(tgrIndexFromReco, process.iterTracking)
+
+    # Now get rid of spurious reference to JetCore step
+    process.earlyGeneralTracks.selectedTrackQuals = cms.VInputTag(
+        cms.InputTag("initialStepSelector", "initialStep")
+        , cms.InputTag("lowPtTripletStepSelector", "lowPtTripletStep")
+        , cms.InputTag("pixelPairStepSelector", "pixelPairStep")
+        , cms.InputTag("detachedTripletStep")
+        , cms.InputTag("mixedTripletStep")
+        , cms.InputTag("pixelLessStepSelector", "pixelLessStep")
+        , cms.InputTag("tobTecStepSelector", "tobTecStep")
+        )
+    process.earlyGeneralTracks.indivShareFrac = cms.vdouble(1.0, 0.16, 0.19, 0.13, 0.11, 0.11, 0.09)
+    process.earlyGeneralTracks.setsToMerge = cms.VPSet(cms.PSet(
+            pQual = cms.bool(True),
+            tLists = cms.vint32(0, 1, 2, 3, 4, 5, 6)))
+
+    process.earlyGeneralTracks.hasSelector = cms.vint32(1, 1, 1, 1, 1, 1, 1)
+
+    process.earlyGeneralTracks.TrackProducers = cms.VInputTag(
+        cms.InputTag("initialStepTracks")
+        , cms.InputTag("lowPtTripletStepTracks")
+        , cms.InputTag("pixelPairStepTracks")
+        , cms.InputTag("detachedTripletStepTracks")
+        , cms.InputTag("mixedTripletStepTracks")
+        , cms.InputTag("pixelLessStepTracks")
+        , cms.InputTag("tobTecStepTracks")
+        )
+
+    # Now get rid of any pre-splitting business
+    process.siPixelClusters = process.siPixelClustersPreSplitting.clone()
+    process.pixeltrackerlocalreco.replace(process.siPixelClustersPreSplitting, process.siPixelClusters)
+    process.pixeltrackerlocalreco.replace(process.siPixelRecHitsPreSplitting, process.siPixelRecHits)
+    process.clusterSummaryProducer.pixelClusters = 'siPixelClusters'
+    process.globalreco.replace(process.MeasurementTrackerEventPreSplitting, process.MeasurementTrackerEvent)
+    process.globalreco.replace(process.siPixelClusterShapeCachePreSplitting, process.siPixelClusterShapeCache)
+
+    # Now restore pixelVertices wherever was not possible with an ad-hoc RunI cfg
+    process.muonSeededTracksInOutSelector.vertices = 'pixelVertices'
```

Comment:
Ciao Slava,
gettig rid of them would have been more work than added value, I fear. As I wrote in the initial comment, if you want to **ignore** them, you can use the earlyGeneralTrack collection, which is produced before running the muon seeded iterations.